crit/Makefile

```diff
@@ -1,8 +1,10 @@
 GO ?= go
 CRIU ?= criu
+CRIT_SRC = $(shell find . -name \*.go)
+CRIT_CLI = cmd/main.go
```

Comment:
I used `=` instead of `?=` since I felt it's reasonable to assume that we won't override this variable, unlike others. The CLI entrypoint will always be consistent, so I didn't see the need for `?=`.

Reply:
Sure, perhaps we should use [simple](https://www.gnu.org/software/make/manual/html_node/Simple-Assignment.html) (`:=`) instead of [recursive](https://www.gnu.org/software/make/manual/html_node/Recursive-Assignment.html) (`=`) assignment?

Reply:
Changed and pushed in #141